CHANGELOG.md

```diff
@@ -0,0 +1,525 @@
+# Changelog
+
+## v5.0.0
+Released by Jordan Harband on 2020-04-24T21:40:59Z
```

Comment:
i'm fine with a date stamp, but i'd like it formatted nicely (YYYY-MM-DD seems good). I don't think we need the name there.

Reply:
I'll try if I can do conditional templating